src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -163,6 +168,10 @@ function createFirstChild(returnFiber, existingChild, newChildren, priority) {
 
 // TODO: This API won't work because we'll need to transfer the side-effects of
 // unmounting children to the returnFiber.
-exports.reconcileChildFibers = function(returnFiber : Fiber, currentFirstChild : ?Fiber, newChildren : ReactNodeList, priority : PriorityLevel) : ?Fiber {
+exports.reconcileChildFibers = function(
+  returnFiber : Fiber, 
+  currentFirstChild : ?Fiber, 
+  newChildren : ReactNodeList, 
+  priority : PriorityLevel) : ?Fiber {
```

Comment:
Just match current style with the colon spacing. We can enable a different lint rule enforcing 1 way or the other later and codemod (it'll be no space before).